datafusion/physical-expr/src/functions.rs

```diff
@@ -433,6 +433,12 @@ pub fn create_physical_fun(
                 execution_props.query_execution_start_time,
             ))
         }
+        BuiltinScalarFunction::CurrentTime => {
+            // bind value for current_time at plan time
+            Arc::new(datetime_expressions::make_current_time(
+                execution_props.query_execution_start_time,
+            ))
```

Comment:
👍 

@mingmwang all the current_time within a query uses the same query_execution_start_time